### PR TITLE
Added changes for User issue

### PR DESCRIPTION
**Pull request by herrone**

### CODE REVIEW

File: Views/UsersPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                };
+
+                userService.Add(newUser);
+                users.Add(newUser);
+
+                // Clear the input fields
+                txtName.Text = string.Empty;
+                txtEmail.Text = string.Empty;
+                pickerAccessLevel.SelectedIndex = -1;
+                adminCredentialsLayout.IsVisible = false;
```
Comment (stefan4h) [inline Views/UsersPage.xaml.cs]: This method violates the SRP as clear inut field should be a seperate method

**Review by stefan4h — CHANGES_REQUESTED** [review]
please change

**Review by stefan4h — APPROVED** [review]

File: Models/User.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using SQLite;
+using System.ComponentModel;
+
+namespace UndacApp.Models
+{
+    public class User : AModel
+    {
+        private string _name;
+        public string Name
```
Comment (Pannoniae) [inline Models/User.cs:9]: This duplicates the Name field in AModel. Please remove.

File: Views/UsersPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        {
+            txtName.Text = string.Empty;
+            txtEmail.Text = string.Empty;
+            pickerAccessLevel.SelectedIndex = -1;
+            adminCredentialsLayout.IsVisible = false;
+        }
+        private async void CheckCredentialsButton_Clicked(object sender, EventArgs e)
+        {
+            // Check credentials
+            if (txtAdminUsername.Text == "systemAdmin" && txtAdminPassword.Text == "password123")
```
Comment (Pannoniae) [inline Views/UsersPage.xaml.cs:46]: Password checking should be handled in a separate method

**Review by Pannoniae — CHANGES_REQUESTED** [review]
Looks good, there are two minor issues. Otherwise, good job!

**Pull request merged**